client/src/components/page/StatsPage.js

```diff
@@ -34,12 +35,18 @@ class StatsPage extends Component {
         return response.json();
       })
       .then(data => {
-        this.setState({ content: data.content });
+        this.setState({ messageString: data.messageString });
+        this.setState({ messageCount: data.messageCount });
```

Comment:
Hard code the message on the client instead of sending it down from the server. The less data going over the network the better the more money we and our users save.